### PR TITLE
feat: [CVS-9299]: Dynatrace Refactoring

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:
Screenshot 2022-03-07 at 8 57 24 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
trigger cypress

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
retrigger standards

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
retrigger jest

File: src/modules/85-cv/pages/health-source/connectors/Dynatrace/components/DynatraceCustomMetrics/DynatraceCustomMetrics.types.ts [hunk 1]
```diff
@@ -12,14 +12,16 @@ export type CreatedMetricsWithSelectedIndex = {
   selectedMetricIndex: number
 }
 
+// createdMetrics, setCreatedMetrics, setMappedMetrics
+
 export interface DynatraceCustomMetricsProps {
-  isFormValid: boolean
+  // isFormValid: boolean
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/connectors/Dynatrace/components/DynatraceCustomMetrics/DynatraceCustomMetrics.types.ts:18]: are these comments needed?

Comment (deepesh-ui) [reply src/modules/85-cv/pages/health-source/connectors/Dynatrace/components/DynatraceCustomMetrics/DynatraceCustomMetrics.types.ts:18]: fixing

**Review by praneshg239 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     "generate-eslint": "node scripts/generate-eslint-config-for-imports.mjs",
     "build-docker": "sh scripts/docker-build.sh",
     "lighthouse": "node scripts/lighthouse/lighthouse.js",
-    "custom-coverage": "npx new-files-coverage-check --e !**/*.test.tsx !**/*.test.ts !**/*.snap !**/*.js !**/*.scss !**/*.mock.ts !**/*.spec.ts !**/*.d.ts !**/*.json !**/*.types.ts !**/*.yml !**/*.yaml !**cypress/** !**cypress/** !**/__test*/** !**/*.png !**/*.svg"
+    "custom-coverage": "npx new-files-coverage-check --e !**/*.test.tsx !**/*.test.ts !**/*.snap !**/*.js !**/*.scss !**/*.mock.ts !**/*.spec.ts !**/*.d.ts !**/*.json !**/*.types.ts !**/*.yml !**/*.yaml !**cypress/** !**/cypress/** !**/__test*/** !**/*.png !**/*.svg"
```
Comment (vkbansal) [inline package.json:52]: LGTM

**Review by vkbansal — APPROVED** [review]
Changes in `package.json` looks good

**Review by vasuki-kalluri — APPROVED** [review]

**Pull request merged**